### PR TITLE
i will never have the money for this

### PR DESCRIPTION
**Pull request by nadav7679**

### CODE REVIEW

File: yaml/nadav/‏‏yamel_exercise2_nadav .yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+models:
+  - model: Audi A8
```
Comment (etayruben) [inline yaml/nadav/‏‏yamel_exercise2_nadav .yaml:2]: you have bad indentation.

**Review by etayruben — DISMISSED** [review]

File: yaml/nadav/‏‏yamel_exercise2_nadav .yaml (hunk 1 above)
Comment (nadav7679) [reply yaml/nadav/‏‏yamel_exercise2_nadav .yaml:2]: y

Comment (nadav7679) [reply yaml/nadav/‏‏yamel_exercise2_nadav .yaml:2]: u stupid haha

**Review by omershilohpsi — APPROVED** [review]

**Review by omershilohpsi — APPROVED** [review]

File: yaml/nadav/‏‏yamel_exercise2_nadav .yaml (hunk 1 above)
Comment (noamu111) [reply yaml/nadav/‏‏yamel_exercise2_nadav .yaml:2]: you are poor

Comment (noamu111) [reply yaml/nadav/‏‏yamel_exercise2_nadav .yaml:2]: you is* poor

Comment (nadav7679) [reply yaml/nadav/‏‏yamel_exercise2_nadav .yaml:2]: I am
Not nice :(

**Review by benedery — APPROVED** [review]

**Pull request merged**